lib/contracts/commit.ts

```diff
@@ -10,6 +10,7 @@ export const commit: ContractDefinition = cardMixins.mixin(
 	slug,
 	name: 'Commit',
 	type,
+	version: '1.0.1',
```

Comment:
If this is technically a breaking change for transformer consumers, should this be `2.0.0`?

Reply:
@joshbwlng It's not a breaking change since it just removes a schema constraint.

Reply:
@codewithcheese Ok, I was assuming it was possibly a breaking change as we'll be removing a field. If it is only removing a constraint and won't cause any consumers to break, then a patch makes sense. (Still haven't caught up with transformers yet, hoping to get to it next week!)